deploy/operator.yaml

```diff
@@ -16,21 +16,10 @@ spec:
       containers:
         - name: devopsconsole-operator
           # Replace this with the built image name
-          image: REPLACE_IMAGE
-          ports:
-          - containerPort: 60000
-            name: metrics
+          image: 172.30.1.1:5000/devopsconsole/devopsconsole-operator:latest
```

Comment:
Is this IP going to be the same everywhere?

Reply:
Yes this is minishift internale registry. 
But note this PR is to show how to deploy operator manually so that you can use CR in _any_ namespace.
We'll do another Pr for e2e test focusing on CI/CD deployment and running e2e test. Most probably this README section will be refactored with some easier makefile target.

Reply:
@corinnekrych I think you don't need to hard code this, you can do `eval $(minishift docker-env)` and then build docker image, and use the same image without pushing it to minishift internal registry, I believe it'll be pain if you are pulling, pushing each time for each change. we are doing the same for toolchain operator.
`

Reply:
you need to push to the internal registry.